README.md

```diff
@@ -14,83 +14,260 @@ A curated list of anime streaming/downloading websites.
 * I am **NOT** promoting the use of free anime websites by any means.
 
 
-## Contents
+# Contents
+* [Anime](##Anime)
+ * [Streaming](###Streaming)
+ * [Downloading](###Downloading)
+  * [DDL](####DDL)
```

Comment:
fix indentation, it appears as a separate point and not under `Downloading`

Reply:
I did not understand it :D

Reply:
If you see it, it looks like this

![image](https://user-images.githubusercontent.com/21009455/93302363-7939f880-f817-11ea-871f-3c804999438d.png)

DDL should be a sub-section under download right?

Reply:
I fixed it wait im gonna commit it rn.